src/css/custom.css

```diff
@@ -122,4 +122,185 @@
     font-size: 0.8rem; /* Smaller font size */
     margin-top: 0.5rem; /* Adjusted spacing */
   }
+}
+
+/* Base logo styling */
+.navbar__brand img {
+  max-width: 200px;
+  height: auto;
+  border-radius: 0 0 20px 20px;
+  background: #fff;
+  padding: 6px 8px;
+  margin-top: -12px;
+  margin-left: 12px;
+  box-shadow: 0px 1px 2px rgba(0, 0, 0, 0.08);
+}
+
+.oss-navbar-logo {
+  height: auto;
+  max-height: 5rem;
+  max-width: 14rem;
+  border-radius: 0 0 1.875rem 1.875rem;
+  background: #fff;
+  padding: 0.625rem;
+  margin-top: -1.25rem;
+  margin-left: 1rem;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.08);
+  transition: all 0.3s ease;
+}
+
+/* Tablet */
+@media (max-width: 768px) {
+  .oss-navbar-logo {
+    max-width: 10rem;
+    max-height: 3.75rem;
+    padding: 0.5rem;
+    margin-top: -1rem;
+  }
+}
+
+/* Phone */
+@media (max-width: 480px) {
+  .oss-navbar-logo {
+    max-width: 8rem;
+    max-height: 3rem;
+    padding: 0.4rem;
+    margin-top: -0.75rem;
+  }
+}
+
+/* Very small phones or zoomed-in browser (e.g. 320px–375px) */
+@media (max-width: 375px) {
```

Comment:
Something a little odd happens at this resolution. This is phone:
![Screenshot 2025-06-14 at 10 56 41 AM](https://github.com/user-attachments/assets/8f578692-987a-4a0d-83a6-db925db7080d)
and this is very small phone: 
![Screenshot 2025-06-14 at 10 57 31 AM](https://github.com/user-attachments/assets/d46286dc-e14b-4a9b-8ca5-65935c9a4a5b)

On very small phone it shrinks too much. The logo should sit in the middle of the navbar like it does in the regular phone. I don't know that we need it to shrink further. Am i missing something?